### DISCUSSION TITLE
本地存储list为什么不显示隐藏目录文件？

### DISCUSSION THREAD

**Discussion by jmperdev** [AlistGo/alist#2078] [General]
for _, f := range rawFiles {
		if strings.HasPrefix(f.Name(), ".") {
			continue
		}
         ...

https://github.com/alist-org/alist/blob/02df3759df423f7c9a9d82133df9e328c6c6e179/drivers/local/driver.go#L71

是做什么考虑的？能提个pr把这段代码去掉吗？

**Comment by xhofe** [comment]
可以

**Comment by foxdodo** [comment]
这样隐藏会导致手机端的app比如foldersync无法用webdav连接隐藏目录
